services/graph/pkg/service/v0/users.go

```diff
@@ -1014,3 +986,34 @@ func isValidUserType(userType string) bool {
 
 	return false
 }
+
+func (g Graph) searchOCMAcceptedUsers(ctx context.Context, odataReq *godata.GoDataRequest) ([]*libregraph.User, error) {
+	users := []*libregraph.User{}
+	if g.config.IncludeOCMSharees {
+		gwc, err := g.gatewaySelector.Next()
+		if err != nil {
+			return nil, errorcode.New(errorcode.GeneralException, err.Error())
+		}
+		term, err := identity.GetSearchValues(odataReq.Query)
+		if err != nil {
+			return nil, errorcode.New(errorcode.InvalidRequest, err.Error())
+		}
+
+		remoteUsersRes, err := gwc.FindAcceptedUsers(ctx, &invitepb.FindAcceptedUsersRequest{Filter: term})
+		if err != nil {
+			// TODO grpc FindAcceptedUsers call failed
```

Comment:
Comment unclear. What is the TODO here?

Reply:
Hm, good question. I just move that code from another place. Let me check ...

Reply:
I've changed to now use our generic CS3 Status to graph mapper. And added a bit of log. I think this is ok.